### ISSUE TITLE
[BUG] - Lidarr - Are download paths hard coded?

### ISSUE THREAD

**Issue by Kamology** [RandomNinjaAtk/arr-scripts#233]
**Application**
Lidarr

**Host platform**
Unraid

**Script**
Lidarr Extended

**Script Version**
Latest release as of 04/10/2024

**Describe the bug**
I'm setting the download path to `/data/processing/downloads/music/incomplete` and `/complete` but the path being used is `/data/processing/downloads/music/incomplete/audio/incomplete` (behavior also exists when using `/mnt/user/media` prefix, which `/data` is a container alias for)

**To Reproduce**
1. set path in extended.conf
2. run script

**Expected behavior**
Downloads reflect path set in extended.conf

**Logs/Screenshots**
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/91dc7ebf-1ad3-4616-b68a-03c2bad435f6)
![Screenshot 2024-04-09 014450](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/a8bfd810-2f23-4522-bed7-692c1e836a2e)
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/87d53d6f-2e2e-47d6-a7f4-3287c0f7832d)
![Screenshot 2024-04-09 014419](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/773b252f-7fd2-4d29-b3e5-cf7a18997399)

TLDR `/audio/incomplete` is being appended to the download path I set

**Comment by Kamology** [comment]
This might be unrelated and a separate configuration error or something but my downloads are also like... phantom downloading it seems. The script logs appear to be downloading stuff but they're not actually appearing in the directory. Idk where they're going/what's happening to them. They're not in the default appdata folder either lol. Idk if these errors are related but they're the only thing I see:

![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/e3f7b833-5862-42aa-81c8-5b89b1f13269)
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/6eaf86b8-8436-4662-9638-29132c2f49b5)

I *did* have successful downloads and imports when using the stock path settings so I'd hope it's not something I did wrong. The reason I want the paths changed from default is so I can have atomic moves (not needing to move downloaded songs from appdata -> my media folder)

**Comment by Kamology** [comment]
Sorry I keep adding to this, but it also appears the "auto config" ignores any custom paths too and just configures with the default paths. These are fresh paths I just made and the fresh download folder config it made: 

![Screenshot 2024-04-10 024328](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/d8f81ae0-26c7-4abf-ba06-1a2b3d6792da)
![Screenshot 2024-04-10 024315](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/65a6c9e8-90be-4ce4-a17a-44b064939d00)

**Comment by RandomNinjaAtk** [comment]
So it was hardcoded and I just fixed that, however... The script has no way to update an existing client that was already added to Lidarr. So once the client has been added, it's whatever path it was originally set at.

If using queuecleaner, it should only at most ever be a temporary location.

**Comment by Kamology** [comment]
Sweet thank you. One extra question.

"If using queuecleaner, it should only at most ever be a temporary location."

Could you clarify what you mean by that? Is "it" in this case the download/import paths? What does queue cleaner do being on vs off and when would you use it/not use it? Just wanna gain a little better understanding of how these scripts work and how I should be configuring them.
